core/chaincode/chaincode_support.go

```diff
@@ -42,7 +43,7 @@ type Runtime interface {
 
 // Launcher is used to launch chaincode runtimes.
 type Launcher interface {
-	Launch(ccid string) error
+	Launch(ccid string, streamHandler extcc.StreamHandler) error
```

Comment:
We could simplify this by using the custodian shim here too.

Reply:
Assuming I didn't misunderstand, the custodian shim is an adapter for the Launcher interface.. we cannot change the adapted ?

Reply:
Ah, nevermind this comment, my mistake.